### PR TITLE
Add flow call to confirm msisdn change

### PR DESCRIPTION
**Pull request by erikh360**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -53,7 +53,6 @@ module.exports = function (grunt) {
                 ],
                 ussd_chw_rapidpro: [
                     'src/index.js',
-                    'src/engage.js',
```
Comment (erikh360) [inline Gruntfile.js:56]: Unrelated changes in the gruntfile that I think was left out of the previous PR

**Comment by DevChima** [comment]
+1
